src/main/java/webserver/servlet/UserListController.java

```diff
@@ -1,25 +1,29 @@
 package webserver.servlet;
 
+import com.google.common.base.Strings;
 import db.DataBase;
 import java.io.IOException;
 import java.util.Collections;
+import javax.servlet.http.HttpSession;
 import utils.HandlebarsTemplate;
 import webserver.domain.ContentType;
-import webserver.domain.Cookies;
 import webserver.domain.HttpHeader;
+import webserver.domain.Sessions;
 import webserver.request.HttpRequest;
 import webserver.response.HttpResponse;
 
 public class UserListController implements Controller {
 
-    private static final String COOKIE_NAME_LOGINED = "logined";
+    public static final String KEY_LOGINED = "logined";
 
     @Override
     public void doGet(HttpRequest httpRequest, HttpResponse httpResponse) {
-        Cookies cookies = httpRequest.getCookies();
-        String loginCookie = cookies.get(COOKIE_NAME_LOGINED);
+        String sessionId = httpRequest.getSessionId();
+        HttpSession session = Sessions.INSTANCE.get(sessionId);
```

Comment:
```suggestion
        HttpSession session = httpRequest.getSession();
```
위와 같이 제공하지 않은 이유가 있을까요 ㅎㅎ

Reply:
Request 안에서 Sessions 싱글톤을 의존하는게 어색한가 싶었는데, 정리하고 보니 그렇지 않은 것 같네요 ㅎㅎ;;
부근을 같이 정리해보았습니다!!